docker-compose.yaml

```diff
@@ -0,0 +1,23 @@
+version: '3.6'
+services:
+  mongo:
+    image: mongo:4.2.6
+    restart: always
+    ports:
+      - 27017:27017
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: root
+      MONGO_INITDB_ROOT_PASSWORD: abcd1234
+  mongo-express:
+    image: mongo-express
+    restart: always
+    ports:
+      - 8081:8081
+    environment:
+      ME_CONFIG_MONGODB_ADMINUSERNAME: root
+      ME_CONFIG_MONGODB_ADMINPASSWORD: abcd1234
```

Comment:
comment - why mongo?

Reply:
it can be used to store results / state - it's optional in the framework, but if we wanted to see all previous tasks that have been executed in last 7 days for example, we can store it there.